packages/containers/src/renderers.js

```diff
@@ -0,0 +1,31 @@
+import { api } from '@talend/react-cmf';
+import Action from './Action';
+import ActionBar from './ActionBar';
+import ActionButton from './ActionButton';
+import ActionDropdown from './ActionDropdown';
+import Actions from './Actions';
+import ActionSplitDropdown from './ActionSplitDropdown';
+
+const renderers = {
+	Action,
+	ActionBar,
+	ActionButton,
+	ActionDropdown,
+	Actions,
+	ActionSplitDropdown,
+};
+
+export default function getRenderers() {
+	Object.keys(renderers).forEach((key) => {
+		try {
+			const component = api.component.get(key);
+			if (component && renderers[key] && renderers[key] !== component) {
+				renderers[key] = component;
+			}
+		} catch (error) {
+			/* empty catch because this just means this component
```

Comment:
I think we could use this to add more contextual information about the non registrered component

Reply:
Could we use an invariant or something.